### PR TITLE
Develop/export bs xml

### PR DESCRIPTION
**Pull request by feuerbutter**
put the 4 parameters in the BergerSehgalRESPXSec2014.cxx vector current form factor calculation into BergerSehgalRESPXSec2014.xml . they are now called fCv3,4,51,52

### CODE REVIEW

**Review by mroda88 — CHANGES_REQUESTED** [review]
- add parameters description in the xml file top table.
- Expose also the hardcoded CA5_0 (from the axial current)

**Comment by idkakorin** [comment]
For uniformity, it is necessary to do the same in the MKSPPPXSec2020 model.

**Review by idkakorin — CHANGES_REQUESTED** [review]
https://github.com/GENIE-MC/Generator/pull/352#issuecomment-1788805265

File: src/Physics/Resonance/XSection/MKSPPPXSec2020.cxx
Comment (idkakorin) [inline src/Physics/Resonance/XSection/MKSPPPXSec2020.cxx:1]: LGTM

File: src/Physics/Resonance/XSection/MKSPPPXSec2020.h
Comment (idkakorin) [inline src/Physics/Resonance/XSection/MKSPPPXSec2020.h:1]: LGTM

**Review by idkakorin — APPROVED** [review]

**Review by mroda88 — APPROVED** [review]
Great

**Pull request merged**